components/org.wso2.carbon.config/src/test/java/org/wso2/carbon/config/configprovider/ConfigProviderFactoryTest.java

```diff
@@ -36,25 +37,20 @@
  * @since 1.0.0
  */
 public class ConfigProviderFactoryTest {
-    public static final String SYS_KEY_MASTER_KEY_FILE = "master.key.file";
-    public static final String SYS_KEY_SEC_PROP_FILE = "sec.prop.file";
-    public static final String SYS_KEY_KEYSTORE_FILE = "keystore.file";
+    private static final String SYS_KEY_MASTER_KEY_FILE = "master.key.file";
+    private static final String SYS_KEY_SEC_PROP_FILE = "sec.prop.file";
+    private static final String SYS_KEY_KEYSTORE_FILE = "keystore.file";
     private static final String PASSWORD = "n3wP4s5w0r4";
+    private static final String OS_NAME_KEY = "os.name";
```

Comment:
I see that both the test cases have defined these constants within the same package. Can we move them to a common constant class?